h2o-core/src/main/java/hex/Model.java

```diff
@@ -355,6 +355,11 @@ public long getOrMakeRealSeed(){
      */
     public String _export_checkpoints_dir;
 
+    /**
+     * Bins for Gains/Lift table, if applicable. Ignored if G/L are not calculated.
+     */
+    public int _gainslift_bins;
```

Comment:
Exposing it as a parameter set before the actual model training has the best value/time invested ratio. G/L are calculated as a part of model driver's logic. The G/L calculations uses vecs not available after the training is done.